src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -399,6 +401,12 @@ def set_feedback(feedback):
     _session.feedback = feedback[:512]
 
 
+@decorators.suppress_all_exceptions()
+def set_recommendation_properties(api_version, recommendation_properties):
+    _session.recommendation_properties = {"api_version": api_version,
+                                          "recommendation_properties": recommendation_properties}
```

Comment:
Please add some comments for this method and properties

Reply:
Already modified in https://github.com/Azure/azure-cli/pull/25875/commits/e676ac4148dd458e82208f27f103239732b98523